docs/examples/02_description_models/03_robot_instances.rst

```diff
@@ -0,0 +1,56 @@
+********************************************************************************
+RobotModel Packages
+********************************************************************************
+
+.. rst-class:: lead
+
+COMPAS Fab provides several ready-to-use Robot packages that can be used for
+demonstrating the capabilities of the package. These packages can be accessed from the
+`compas_fab.robots.RobotLibrary` class. The robot packages are loaded from local data
+files located in the folder `src/compas_fab/data/robot_library`. They contain
+the `robot.model`, `robot.semantics` and meshes associated with the robot links.
+
+The robot packages can be used with the PyBullet Planning backends for planning purpose and
+without backend for visualization purpose. The robot packages will be extended in the future
+to work with the ROS MoveIt planning backend.
```

Comment:
I wouldn't write anything about ROS/MoveIt, because we don't really know if this is possible at all.

Reply:
```suggestion
```